Packs/MicrosoftTeams/Integrations/MicrosoftTeams/MicrosoftTeams.py

```diff
@@ -2247,9 +2315,10 @@ def message_handler(integration_context: dict, request_body: dict, channel_data:
                             investigation_id: str = mirrored_channel.get('investigation_id', '')
                             username: str = from_property.get('name', '')
                             user_email: str = get_team_member(integration_context, team_member_id).get('user_email', '')
+                            demisto.debug(f"Adding Entry {message} to investigation {investigation_id}")
                             demisto.addEntry(
                                 id=investigation_id,
-                                entry=message,
+                                entry=message.strip(),
```

Comment:
The original intent was to use `lstrip` to handle unexpected spaces introduced, but `strip` also makes sense.
Please add a short docstring to explain this.
This should be tested via a unit test.